chipsec/cfg/pch_c620.xml

```diff
@@ -55,62 +59,69 @@ XML configuration file for
     <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 
     <!-- Sideband Register Access Registers -->
-    <register name="SBREG_BAR" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0x10" size="4" desc="Sideband Register Access BAR">
+    <register name="SBREG_BAR"  type="pcicfg" device="P2SBC" offset="0x10" size="4" desc="Sideband Register Access BAR">
       <field name="RBA" bit="24" size="8" desc="Register Base Address"/>
     </register>
-
-    <register name="P2SBC" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0xE0" size="2" desc="P2SB Configuration Register">
+    <register name="SBREG_BARH" type="pcicfg" device="P2SBC" offset="0x14" size="4" desc="Sideband Register Access BAR High DWORD">
+      <field name="RBAH" bit="0" size="32" desc="Register Base Address"/>
+    </register>
+    <register name="P2SBC"      type="pcicfg" device="P2SBC" offset="0xE0" size="4" desc="P2SB Configuration Register">
       <field name="HIDE" bit="8" size="1" desc="Hide SBREG_BAR"/>
     </register>
+    <register name="P2SB_HIDE"  type="pcicfg" device="P2SBC" offset="0xE1" size="1" desc="P2SB Configuration Register hide-unhide">
+      <field name="HIDE" bit="0" size="1" desc="Hide SBREG_BAR"/>
+    </register>
 
     <!-- Power Management Controller -->
-    <register name="ABASE" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x40" size="4" desc="ACPI Base Address">
+    <register name="ABASE"       type="pcicfg" device="PMC" offset="0x40" size="4" desc="ACPI Base Address">
+      <field  name="Base" bit="8" size="24" desc="Base Address"/>
       <field name="STYPE" bit="0" size="1" desc="Space Type (always 1 - I/O space)"/>
-      <field name="BA"    bit="8" size="8" desc="Base Address"/>
```

Comment:
I think we need to keep the Base Address around, otherwise we are rendering the accesses to the BAR based registers useless

Reply:
It's there, just named "Base" plus the size attribute is now fixed.   This aligns with what is in common.xml and what it overrides in the CPU xml for this platform.

Reply:
I see that, then what we need to fix is the ABASE bar definition as it is pointing to field BA which is non-existent